src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -447,6 +452,10 @@ public Loan declareItemLost(String comment, DateTime dateTime) {
     return this;
   }
 
+  public boolean isDeclaredLost() {
+    return getAction() == DECLARED_LOST;
+  }
```

Comment:
Why is the action history of the loan being used to determine whether an item has been declared lost?

Reply:
it is not action history, it is current value of `loan.action` property. 

Is there any different way to find out if a loan is declared lost? Check item status?

Reply:
> Is there any different way to find out if a loan is declared lost? Check item status?

What does the story say to use for this decision?

My initial reaction is that it would be the item status that decided this, not the loan action history (that the latest action is present on the loan is an unfortunate effect of the way this was designed).

Reply:
There are no particular steps in jira. I'll use item status then.

Reply:
Please can you ask Holly on the issue to confirm this decision. I think we need this explicitly deciding and documenting.

Reply:
Asked Emma in jira.

Reply:
Emma has confirmed that item.status is enough to confirm that loan is declared lost.